README.md

```diff
@@ -28,6 +28,28 @@ Coming soon.
       roles:
         - kostiantyn-nemchenko.patroni
 
+## Example Inventory
+
+      [all]
+      patroni1 ansible_host=192.168.88.160
+      patroni2 ansible_host=192.168.88.159
+      patroni3 ansible_host=192.168.88.158
+
+      [patroni]
+      patroni1
+      patroni2
+      patroni3
+
+      [etcd]
+      patroni1
+      patroni2
+      patroni3
+
+      [etcd-master]
```

Comment:
What's the reason to add this group here? The retr0h.etcd role suggested as dependency works without it.

Reply:
playbook retr0h.etcd wont work

Reply:
Ok, I will take a look at it to get more details. Thank you